x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -80,7 +75,7 @@ export const AllCasesList = React.memo<AllCasesListProps>(
       ...DEFAULT_FILTER_OPTIONS,
       ...initialFilterOptions,
     });
-    const [queryParams, setQueryParams] = useState<QueryParams>(DEFAULT_QUERY_PARAMS);
+    const { queryParams, setUrlQueryParams } = useUrlState(isSelectorView);
```

Comment:
If I'm viewing cases inside the Security app, and then switch to the Alerts page and select 'Add to existing case' for an alert, I am seeing this error. This doesn't happen inside Observability.

<img width="938" alt="image" src="https://user-images.githubusercontent.com/7405507/199798867-034c8bcc-a67c-4151-9b8a-d5744d1baa43.png">

Reply:
@MadameSheema It seems that there is no cypress test that test attaching an alert to a case. I think we should add one so the CI can catch these type of bugs (we change something in cases and breaks the security solution integration). What do you think?

Reply:
This should be fixed now! I didn't know this page so didn't test it manually 😅